Catappult/appcoins-sdk-ios
# iOS Attribution

**What does this PR do?**

   This PR adds a service that sends a request to AppTech endpoint,  receives an Attribution and stores the guestUID Attribution's attribute.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] AppcSDKInternal.swift
- [ ] AttributionRepository.swift
- [ ] AttributionClient.swift

**How should this be manually tested?**

  Run Trivial Drive with this SDK and verify if the service returns the AttributionRaw

**What are the relevant tickets?**

  Tickets related to this pull-request: APP-2628

**Questions:**

   Does this add new dependencies which need to be added? No

**Code Review Checklist**
